src/manage_nsfs/manage_nsfs_validations.js

```diff
@@ -16,6 +15,9 @@ const { throw_cli_error, get_config_file_path, get_bucket_owner_account,
 const { TYPES, ACTIONS, VALID_OPTIONS, OPTION_TYPE, FROM_FILE, BOOLEAN_STRING_VALUES,
     GLACIER_ACTIONS, LIST_UNSETABLE_OPTIONS } = require('../manage_nsfs/manage_nsfs_constants');
 
+const valid_access_key_regex = /^[a-zA-Z0-9]+$/;
+const valid_secret_key_regex = /^[a-zA-Z0-9+/]+$/;
```

Comment:
I would move these regex to be in a common module like string_utils where we have the other common regex defined.
Also it is possible to set the exact length in the regex so it doesn't require more checks.
```suggestion
const valid_access_key_regex = /^[a-zA-Z0-9]{20}$/;
const valid_secret_key_regex = /^[a-zA-Z0-9+/]{40}$/;
```

Reply:
nice! will do